### ISSUE TITLE
Make Jaws installable from crates.io

### ISSUE THREAD

**Issue by jhawksley** [jhawksley/rjaws#1]
It should be possible to install Jaws from https://www.crates.io, rather than having to build it oneself.

https://doc.rust-lang.org/book/ch14-04-installing-binaries.html

**Comment by jhawksley** [comment]
This issue implies CICD too.
